src/Components/TicketSelector/Calendar/Calendar.jsx

```diff
@@ -0,0 +1,77 @@
+import React, { useState, useEffect, useRef } from "react";
+import DatePicker, { registerLocale } from "react-datepicker";
+import ko from "date-fns/locale/ko";
+import { useClickOutside } from "../../Hooks/clickOutside";
+import "react-datepicker/dist/react-datepicker.css";
+import "./customStyle.scss";
+
+registerLocale("ko", ko);
+
+const Calendar = (props) => {
+  const [startDate, setStartDate] = useState(props.startDate);
+  const [endDate, setEndDate] = useState(null);
+  const calendarRef = useRef(null);
+
+  const handleChange = (dates) => {
+    const [start, end] = dates;
+    setStartDate(start);
+    setEndDate(end);
+  };
```

Comment:
구조분해 할당 잘 써주셨습니다! 인자로 들어오는 것이 dates라는 걸 보여준다는 점에서 그대로 남겨둬도 좋지만 직접해줄 수도 있겠네요.
```suggestion
  const handleChange = ([startDate, endDate]) => {
    setStartDate(startDate);
    setEndDate(endDate);
  };
```
혹은 시작일, 종료일을 하나의 객체로 묶어 조금 더 간단히 관리할 수도 있겠고요.
```suggestion
  const handleChange = ([startDate, endDate]) => setDates({ startDate, endDate })
```

Reply:
omg~~ 배워갑니다 종택닙...